cmake/faceliftMacros.cmake

```diff
@@ -975,15 +958,13 @@ function(facelift_add_qml_plugin PLUGIN_NAME)
     file(WRITE ${CMAKE_BINARY_DIR}/${INSTALL_PATH}/qmldir "${QMLDIR_CONTENT}")
     file(WRITE ${CMAKE_BINARY_DIR}/${INSTALL_PATH}/qmldir.installed "${QMLDIR_CONTENT}")
 
-    if(NOT CMAKE_CROSSCOMPILING AND NOT WIN32)
-        # not supported for now on Win32 since the required libraries can't be loaded without setting the PATH variable
-        add_custom_command(
-            OUTPUT  ${CMAKE_BINARY_DIR}/${INSTALL_PATH}/plugins.qmltypes
-            COMMAND ${_qt5Core_install_prefix}/bin/qmlplugindump -noinstantiate ${URI} ${PLUGIN_MAJOR_VERSION}.${PLUGIN_MINOR_VERSION} ${CMAKE_BINARY_DIR}/imports -output ${CMAKE_BINARY_DIR}/${INSTALL_PATH}/plugins.qmltypes || touch ${CMAKE_BINARY_DIR}/${INSTALL_PATH}/plugins.qmltypes
-            DEPENDS ${PLUGIN_NAME}
-        )
-        add_custom_target("generate_qmltypes_${PLUGIN_NAME}" ALL DEPENDS ${CMAKE_BINARY_DIR}/${INSTALL_PATH}/plugins.qmltypes)
-        install(FILES ${CMAKE_BINARY_DIR}/${INSTALL_PATH}/plugins.qmltypes DESTINATION ${INSTALL_PATH})
-    endif()
+    add_custom_command(
```

Comment:
Is it safe to remove NOT CMAKE_CROSSMOPLILING?

Reply:
it was introduced for WIN32
> not supported for now on Win32

wasn't it ?

Reply:
I did not check TBH. However there is a logical AND, therefore if we drop the windows support, we have it always true, the only part left for checking is CMAKE_CROSSCOMPILING. We can still cross compile it for e.g. Arm or other embedded. If you prove this changes nothing, I approve PR.

Reply:
agree.
I could delete only "NOT WIN32" here, let CMAKE_CROSSCOMPILING be